utilities/test_suite/README.md

```diff
@@ -127,6 +127,10 @@ python runTests.py --case_start 0 --case_end 89 --test_type 0 --qa_mode 1 --batc
 ``` python
 python runTests.py --case_list 21 36 63 --test_type 1 --qa_mode 1 --batch_size 8 --num_runs 100
 ```
+-   QA mode (Performance tests) - Tolerance based PASS/FAIL tests for RPP HIP/HOST functionalities checking achieved improvement in performance percentage over BatchPD versions after comparison to a threshold percentage of improvement
+``` python
+python runTests.py --case_list 21 36 63 --test_type 1 --qa_mode 1 --batch_size 8 --num_runs 100
```

Comment:
Same lines are added with merge
Please delete it

Reply:
Done